src/analysis/convex/side.lean

```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Joseph Myers
 -/
 import analysis.convex.between
-import analysis.convex.topology
+import analysis.convex.normed
```

Comment:
I have some unfinished changes to make the topological lemmas in this file work for a general real topological vector space without requiring norms. My notes say the missing piece was an instance to deduce `has_continuous_vadd` for an affine subspace from `has_continuous_vadd` for the whole space (which should be straightforward, though I'm not sure where such an instance should go).

Reply:
I.e. I had `[add_comm_group V] [module ℝ V] [topological_space V] [topological_add_group V] [has_continuous_smul ℝ V] [add_torsor V P] [topological_space P] [has_continuous_vadd V P] {s : affine_subspace ℝ P}` and needed an instance for `has_continuous_vadd s.direction s`.